src/router/tourists/index.ts

```diff
@@ -63,4 +76,11 @@ touristsRouter.post(
   }
 );
 
+// Utility function
+async function fetchBase64FromUrl(url: string): Promise<string> {
+  const response = await fetch(url);
```

Comment:
This won't work. You are making a request. You need to research more about how this API works, make more testing to see whats inside this value, and look how you can retrieve the image.